src/css/components/_graphic-with-text-buttons.scss

```diff
@@ -10,6 +10,7 @@
     width: 100%;
     display: flex;
     align-items: center;
+    justify-content: center;
 }
 
 .graphic-with-text-buttons-template .large-graphic {
```

Comment:
Noticing a slight size difference between normal and templated images

![Screenshot from 2022-04-01 13-32-44](https://user-images.githubusercontent.com/10703941/161314395-46e356ba-43d1-422b-b44d-636be5091eb3.png)
![Screenshot from 2022-04-01 13-36-17](https://user-images.githubusercontent.com/10703941/161314401-6378f32f-5be2-4dcc-8b82-70b32f55a99c.png)

The normal image seems to have an inline style applied(`style="height: auto; width: 100%"`) while the templated image does not.

**NOTE:** This issue also seems to apply to `TEXTBUTTONS_WITH_GRAPHIC`, `TEXT_WITH_GRAPHIC`, `GRAPHIC_WITH_TILES` and `TILES_WITH_GRAPHIC`